test/scripts/lint_fix.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Please do not copy code, but make the original `lint.sh` script take a parameter.

Reply:
Implemented, but I haven't worked with parameterized bash scripts before - please double-check me?